update_version.sh

```diff
@@ -0,0 +1,68 @@
+# #!/bin/bash
```

Comment:
interesting, this should be something we add to all of our repos. Nice enhancement